Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -1013,6 +1028,7 @@ private void analysis_setSubscriptions() {
 
       final Map<String, List<String>> subscriptions = new THashMap<String, List<String>>();
       subscriptions.put(AnalysisService.HIGHLIGHTS, myVisibleFiles);
+      subscriptions.put(AnalysisService.IMPLEMENTED, myVisibleFiles);
```

Comment:
OK, I see it too. I was not able to reproduce because DAS based implemented markers were commented out in `plugin.xml`.

The following statement from the DAS spec causes the problem: `It is an error if any of the keys in the map are not valid services. If there is an error, then the existing subscriptions will remain unchanged.`

@bwilkerson 
This seems not very client friendly and backward (or forward?) compatible.
Maybe we could subscribe clients for notifications DAS understands, and send back a list of services which DAS did not understand.